api/src/main/java/org/apache/cloudstack/api/response/UserVmResponse.java

```diff
@@ -124,6 +124,10 @@
     @Param(description = " an alternate display text of the template for the virtual machine")
     private String templateDisplayText;
 
+    @SerializedName("deployasistemplate")
```

Comment:
You want to move this to `ApiConstants....`

Reply:
Done